src/modules/10-common/navigation/ProjectSetupMenu/__tests__/ProjectSetupMenu.test.tsx

```diff
@@ -24,7 +24,11 @@ jest.mock('framework/PreferenceStore/PreferenceStoreContext')
   }
 })
 
-jest.mock('services/cd-ng')
+jest.mock('services/cd-ng', () => ({
+  useGetSettingValue: jest.fn().mockImplementation(() => {
+    return { data: { data: { value: 'false' } } }
+  })
```

Comment:
100% not a blocker, but you can simplify with these things using `mockReturnValue`
```suggestion
  useGetSettingValue: jest.fn().mockReturnValue({ data: { data: { value: 'false' } } })
```